orchagent/intfsorch.cpp

```diff
@@ -177,6 +177,31 @@ void IntfsOrch::decreaseRouterIntfsRefCount(const string &alias)
                   alias.c_str(), m_syncdIntfses[alias].ref_count);
 }
 
+bool IntfsOrch::setRouterIntfsMpls(Port &port)
```

Comment:
add const qualifier if it's not being updated

Reply:
Done